docs/cli-reference/ark_server.md

```diff
@@ -14,10 +14,14 @@ ark server [flags]
 ### Options
 
 ```
-  -h, --help                     help for server
-      --log-level                the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
-      --metrics-address string   the address to expose prometheus metrics (default ":8085")
-      --plugin-dir string        directory containing Ark plugins (default "/plugins")
+      --backup-sync-period duration               how often to ensure all Ark backups in object storage exist as Backup API objects in the cluster
+  -h, --help                                      help for server
+      --log-level                                 the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
+      --metrics-address string                    the address to expose prometheus metrics (default ":8085")
+      --plugin-dir string                         directory containing Ark plugins (default "/plugins")
+      --restic-timeout duration                   how long backups/restores of pod volumes should be allowed to run before timing out
+      --restore-only                              run in a mode where only restores are allowed; backups, schedules, and garbage-collection are all disabled
```

Comment:
https://github.com/heptio/ark/blob/master/docs/use-cases.md#L22 will need to be updated to reflect this change.